more_itertools/more.py

```diff
@@ -545,3 +548,47 @@ def _get_values(self, value):
 
     def __getitem__(self, value):
         return self._get_values(value)
+
+
+def look_ahead(iterable, n=1):
```

Comment:
Slight preference for the name `spy` for brevity and similarity with `peek`. Feel free to overrule.